### PR TITLE
Add dating matched card

### PR DESCRIPTION
**Pull request by zkhin**
https://real-social.atlassian.net/browse/BE-151

### CODE REVIEW

File: real-main/app/models/card/templates.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def __init__(self, user_id):
 class ChatCardTemplate(CardTemplate):
 
     action = 'https://real.app/chat/'
-    notify_user_after = pendulum.duration(seconds=5)
+    notify_user_after = pendulum.duration(minutes=5)
```
Comment (mfogel) [inline real-main/app/models/card/templates.py:22]: Was this change intentional? I think this will affect how long before someone gets notified they have an unread message in chat

Comment (zkhin) [reply real-main/app/models/card/templates.py:22]: Yeah, this was changed to `5 seconds` (origin was 5 minutes) for testing the APNS issue and forgot to roll back the duration.

Comment (mfogel) [reply real-main/app/models/card/templates.py:22]: ok, cool
